KsApi/models/Backing.swift

```diff
@@ -91,8 +114,27 @@ extension Backing: EncodableType {
   }
 }
 
+extension Backing.PaymentSource: Argo.Decodable {
```

Comment:
I think we could have this only conform to Swift's `Decodable` and do a similar function like before to return an `Argo.Decodable` `success` or `failure`?

Reply:
Just a thought, perhaps this PR is not the place to start doing that, but I think that can be an approach to help us to start moving away from Argo.